data/human/hails.txt

```diff
@@ -2155,13 +2491,16 @@ phrase "friendly disabled"
 		"Our ship took quite the pounding!"
 		"Hey!"
 		"S.O.S! S.O.S!"
+		"Mayday! Mayday!"
+		"We're going under!"
```

Comment:
Spaceships don't go under (they crash, or they are adrift). I wouldn't expect a space-ship captain to say this.

Reply:
They could just be using a metaphor.

Reply:
Yes, I understood that. It still sounds to me as something a space-ship captain would not say, but I don't have very strong objections to it either. Feel free to leave the text in.